cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -535,6 +535,30 @@
         /var/snap/{,*/} r,
         /var/snap/*/**/ rw,
 
+        # Allow creating placeholder directory in /tmp/.snap/ as support for
+        # the writable mimic code that can poke holes in arbitrary read-only
+        # places using tmpfs and bind mounts.
+        /tmp/ r,
+        /tmp/** rw,
```

Comment:
This seems to be more generous than the comment above?

Reply:
Thanks, I will see if we can be more specific.

Reply:
I agree with Pawel, either the comment needs an update for the rule (the later is of course prefered :)

Reply:
I made the rule more strict now.